src/core/BUILD

```diff
@@ -4460,6 +4460,7 @@ grpc_cc_library(
     language = "c++",
     deps = [
         "channel_args",
+        "grpc_lb_subchannel_list",
```

Comment:
This dependency should not be needed.  Only leaf policies need to use the subchannel_list library.